project/BuildInfoSettings.scala

```diff
@@ -47,7 +47,8 @@ object BuildInfoSettings {
       "commitDate" -> commitDate,
       "ciBuild" -> ciBuild,
       "ciTag" -> ciTag,
-      "gitTag" -> gitTag
+      "gitTag" -> gitTag,
+      "oldestSupportedVersion" -> "test-tag-0"
```

Comment:
@fm3 `test-tag-0` is the actual commit tag (`git describe --abbrev=0 --tags`). Should I change `      "oldestSupportedVersion" -> "test-tag-0"` to `      "oldestSupportedVersion" -> "18.08.0"` anyway?